### PR TITLE
Chore: Remove wheel from requirements.txt

### PR DESCRIPTION
**Pull request by JamesHollyer**
## Motivation for features / changes
This package is no longer needed in newer versions of python. It comes as a seed package where needed. It is even causing some users problems(#6567)

### CODE REVIEW

**Review by AnandInguva — APPROVED** [review]
Thanks

**Review by arcra — APPROVED** [review]
LGTM, but please sanity check before merging that we can create the pip package without this explicit dependency, or else, update our internal release documentation if any changes are necessary for that process.

**Comment by JamesHollyer** [comment]
Building the MPM requires a bit of work ATM. I am about to do that work for a patch release. In an effort to not duplicate work, I am going to merge this in and if I run into problems during the patch release I may revert it.

**Pull request merged**
